src/ralph/export_to_ng/resources.py

```diff
@@ -427,8 +445,61 @@ def get_queryset(self):
         return list(chain(parents, children))
 
 
+class NetworkKindResource(resources.ModelResource):
+    class Meta:
+        fields = ('id', 'name',)
+        model = models_network.NetworkKind
+
+
+class DiscoveryDataCenterResource(resources.ModelResource):
```

Comment:
I think this docsting is obsolete.
Something more like this
```
This exports only data centers from:
models_network.DataCenter
```
is correct but in such case there is no point to put it at all, imho.